### PR TITLE
Implement remaining script API commands

### PR DESCRIPTION
**Pull request by rfmejia**
Addresses remaining items in #158.

Please check if tests for these commands are satisfactory, I could only come up with a basic test for `scriptDebug`, and for `scriptKill` I couldn't figure out how to fork a long-running script (e.g., infinite loop) and attempt to kill it.

### CODE REVIEW

**Comment by anatolysergeev** [comment]
Hello, you can try to check ideas from previous try of implementation of this commands.
And there are comments where we decided to postponed their implementation without proper tests.
https://github.com/zio/zio-redis/pull/269#discussion_r555954399
And here you can see my implementation of tests, that is going to test
https://github.com/zio/zio-redis/pull/269/commits/b68e9d5988d43e5ef973d22e3a6c8f5112f6cccd

May be that can help you

File: redis/src/test/scala/zio/redis/ScriptingSpec.scala [hunk 1]
```diff
@@ -179,6 +188,28 @@ trait ScriptingSpec extends BaseSpec {
...
+          val lua2 = """return "2""""
+          for {
+            sha1 <- scriptLoad(lua1)
+            sha2 <- scriptLoad(lua2)
+            _    <- scriptFlush()
+            res  <- scriptExists(sha1, sha2)
+          } yield assertTrue(res == Chunk(false, false))
+        }
+      ),
+      suite("scriptKill")(
```
Comment (anatolysergeev) [inline redis/src/test/scala/zio/redis/ScriptingSpec.scala:204]: There are no success test(

**Comment by rfmejia** [comment]
Thanks for the review, I checked the linked PR and I tried to perform the same test as you did (infinite loop in Lua and test for `scriptKill` success) and got the same non-terminating forked fiber issue. I'll investigate this further.

**Comment by mijicd** [comment]
Implemented in #750

**Pull request closed without merging**